include/laser_assembler/base_assembler.h

```diff
@@ -56,7 +58,7 @@ namespace laser_assembler
 /**
  * \brief Maintains a history of point clouds and generates an aggregate point cloud upon request
  */
-template<class T>
+template<class T, class V>
```

Comment:
that breaks the API too unforttunately, you probably need to create another base class from which BaseAssembler would depend on